src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Tabs/basic.html.twig

```diff
@@ -41,7 +41,17 @@
       {{ form_widget(productForm.manufacturer) }}
     </div>
     <div class="col-md-3 right-column">
+      {{ include('@PrestaShop/Admin/Common/unavailable-feature.html.twig', {
+        'featureTitle': 'Reference'|trans({}, 'Admin.Catalog.Feature'),
+        'helpbox': 'Your reference code for this product. Allowed special characters: .-_#.'|trans({}, 'Admin.Catalog.Feature'),
+      }) }}
+
       {{ form_row(productForm.shortcuts) }}
+
+      {{ include('@PrestaShop/Admin/Common/unavailable-feature.html.twig', {
+        'featureTitle': 'Create category'|trans({}, 'Admin.Catalog.Feature'),
```

Comment:
```suggestion
        'featureTitle': 'Create a new category'|trans({}, 'Admin.Catalog.Feature'),
```
This string already exists, maybe we can keep it? Or did you change it on purpose?